www/roster/models/committee.rb

```diff
@@ -20,7 +20,7 @@ def self.serialize(id, env)
     info = JSON.parse(File.read(File.join(comdev, 'projects.json')))[id]
 
     image_dir = ASF::SVN.find('site-img')
-    image = Dir[File.join(image_dir, "#{id}.*")].map {|path| File.basename(path)}.last
+    image = Dir[File.join(image_dir, "#{id}*.{svg,eps,ai,pdf}")].map {|path| File.basename(path)}.first
```

Comment:
Why the change to .first instead of .last?

Reply:
[logos/about.html](http://www.apache.org/logos/about.html) states: `The first logo, lexicographically, will be picked as the default.` Changing this to `.first` will make sure that Whimsy uses the default logo as well.